terraform/discord-mods-bot/_terraform.tf

```diff
@@ -29,6 +29,5 @@ data "terraform_remote_state" "shared" {
 }
 
 provider "aws" {
-  profile = "default"
```

Comment:
I removed profile because it creates issues for me. If you think we should have it and I'm doing something wrong (probably it's the case 😅), let me know 👍